cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -34,7 +34,7 @@ private import Node0ToString
 cached
 private newtype TIRDataFlowNode =
   TNode0(Node0Impl node) { DataFlowImplCommon::forceCachingInSameStage() } or
-  TVariableNode(Variable var, int indirectionIndex) {
+  TVariableNode(GlobalLikeVariable var, int indirectionIndex) {
```

Comment:
Should we also rename this from `TVariableNode` to `T GlobalLikeVariable`?

Reply:
Good idea. Done in 46b47db78ea9581d012fa330352cfe9becdc541f.

I didn't rename the `class` wrapper as this would be a breaking change. We _could_ do that, and then provide a deprecated type synonym, but I'm not sure I want to expose the name `GlobalLikeVariable` anyway

Reply:
Thanks for clarifying this. I was wondering about this. Maybe make the comment on the class a bit more explicit?

Reply:
I've made this more explicit in 4474b5466296f1df00bdb018ccd26c72020eafd2. Is that what you were thinking of?